### PR TITLE
Fix error logged when repos qs is empty

### PR DESCRIPTION
**Pull request by guillep2k**
Fixes invalid error log in the issues list for the user when no specific repositories are requested.

### CODE REVIEW

**Review by techknowlogick — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]

**Comment by techknowlogick** [comment]
please run `make fmt`

File: routers/user/home.go [hunk 1]
```diff
@@ -389,21 +389,23 @@ func Issues(ctx *context.Context) {
...
-		// remove "[" and "]" from string
-		reposQuery = reposQuery[1 : len(reposQuery)-1]
-		//for each ID (delimiter ",") add to int to repoIDs
-		for _, rID := range strings.Split(reposQuery, ",") {
-			// Ensure nonempty string entries
-			if rID != "" && rID != "0" {
-				rIDint64, err := strconv.ParseInt(rID, 10, 64)
-				if err == nil {
-					repoIDs = append(repoIDs, rIDint64)
+	if(reposQuery != "") {
```
Comment (lafriks) [inline routers/user/home.go]:
```suggestion
	if len(reposQuery) != 0 {
```

**Review by lafriks — CHANGES_REQUESTED** [review]

**Comment by guillep2k** [comment]
> 
> 
> please run `make fmt`

Sorry, it's been a while. Fixed by applying @lafriks suggestion.

**Comment by guillep2k** [comment]
@lafriks done.

**Review by techknowlogick — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**